### ISSUE TITLE
Use calc() for variable calculations

### ISSUE THREAD

**Issue by TeyaVes** [telerik/kendo-themes#1395]
**Describe the bug**
Currently, in our variables, we mostly use mathematical operations without calc(), which causes issues if the two variables are in different units, e.g:

```
$checkbox-size: 1em;
$checkbox-border-width: 1px;
$checkbox-line-height: $checkbox-size + $checkbox-border-width !default;
```
 
We should replace these with calc() or addtwo()

**Comment by joneff** [comment]
`add-two` is no longer there, so it's `calc`

**Comment by joneff** [comment]
The task is too broadly defined. As we'll improve on variables gradually, we'll log items where applicable. This one will target only checkboxes / radio

**Comment by joneff** [comment]
Fixed by #1916
